docs/data/material/components/buttons/buttons.md

```diff
@@ -181,3 +181,13 @@ However:
 ```
 
 This has the advantage of supporting any element, for instance, a link `<a>` element.
+
+## Material You version
+
+This component also supports the new Material Design spec - [Material You](https://m3.material.io/). It is implemented in form of a new component in the experimental `@mui/material-next` package.
```

Comment:
Here's how I would phrase this for clarity—I think it's helpful to add some more context:

```suggestion
The default Button component follows the Material Design 2 specs.
For the MD3 ([Material You](https://m3.material.io/)) version, install and import from the experimental `@mui/material-next` package:
```